tests/share/test_packager.py

```diff
@@ -464,3 +464,55 @@ def test_multi_view_non_snapshot_round_trip(self, reset_db: None) -> None:
 
         for n in (4, 0, 8, 10, 6):
             self.__restore_and_check_table(bundles[n], f'replica_{n}')
+
+    def test_replica_ops(self, reset_db: None, clip_embed: pxt.Function) -> None:
+        t = pxt.create_table('test_tbl', {'icol': pxt.Int, 'scol': pxt.String})
+        t.insert({'icol': i, 'scol': f'string {i}'} for i in range(10))
+        v = pxt.create_view('test_view', t)
+        v.add_computed_column(iccol=(v.icol + 1))
+
+        t_bundle = self.__package_table(t)
+        v_bundle = self.__package_table(v)
+
+        clean_db()
+        reload_catalog()
+
+        self.__restore_and_check_table(v_bundle, 'view_replica')
+        # Check that test_tbl was instantiated as a system table
+        assert pxt.list_tables() == ['view_replica']
+        system_path = pxt.catalog.Path('_system', allow_system_paths=True)
+        system_contents = pxt.catalog.Catalog.get().get_dir_contents(system_path)
```

Comment:
I really want us to get away from using module internals in tests, which has created a fair amount of work for me in the past. How about we add a _include_sys_tbls or similar kwarg to list_tables()?